Sources/UberCore/Authentication/Authenticators/BaseAuthenticator.swift

```diff
@@ -25,27 +25,26 @@
 import UIKit
 
 /// Base class for authorization flows
-@objc(UBSDKBaseAuthenticator) public class BaseAuthenticator: NSObject, UberAuthenticating {
+public class BaseAuthenticator: UberAuthenticating {
     /// Scopes to request during login
-    @objc public var scopes: [UberScope]
+    public var scopes: [UberScope]
     
-    @objc public var requestUri: String?
+    public var requestUri: String?
```

Comment:
How about having them as constant properties, assuming that we wouldn't want them to mutate once the instance is created?